### PR TITLE
feat: bump sunodo/sdk version

### PR DESCRIPTION
**Pull request by endersonmaia**

### CODE REVIEW

File: cpp-low-level/Dockerfile [hunk 1]
```diff
@@ -17,13 +17,17 @@ FROM --platform=linux/riscv64 riscv64/ubuntu:22.04
 LABEL io.sunodo.sdk_version=0.1.0
 LABEL io.cartesi.rollups.ram_size=128Mi
 
+ENV MACHINE_EMULATOR_TOOLS_VERSION=0.12.0
 RUN <<EOF
 apt-get update
-apt-get install -y --no-install-recommends busybox-static=1:1.30.1-7ubuntu3
+apt-get install -y --no-install-recommends busybox-static=1:1.30.1-7ubuntu3 wget ca-certificates
```
Comment (tuler) [inline cpp-low-level/Dockerfile:23]: Lock the exact version of installed packages for better reproducibility

File: cpp-low-level/Dockerfile [hunk 2]
```diff
@@ -17,13 +17,17 @@ FROM --platform=linux/riscv64 riscv64/ubuntu:22.04
...
+cd /tmp
+wget https://github.com/cartesi/machine-emulator-tools/releases/download/v${MACHINE_EMULATOR_TOOLS_VERSION}/machine-emulator-tools-v${MACHINE_EMULATOR_TOOLS_VERSION}.tar.gz
```
Comment (tuler) [inline cpp-low-level/Dockerfile:25]: Is it possible to use curl instead, and not save the file and just pipe into tar?

File: cpp-low-level/Dockerfile [hunk 3]
```diff
@@ -17,13 +17,17 @@ FROM --platform=linux/riscv64 riscv64/ubuntu:22.04
 LABEL io.sunodo.sdk_version=0.1.0
 LABEL io.cartesi.rollups.ram_size=128Mi
 
+ENV MACHINE_EMULATOR_TOOLS_VERSION=0.12.0
```
Comment (tuler) [inline cpp-low-level/Dockerfile:20]: ENV or ARG?

Comment (endersonmaia) [reply cpp-low-level/Dockerfile:20]: Yes, ARG would be better to change during build.

File: cpp-low-level/Dockerfile (hunk 2 above)
Comment (endersonmaia) [reply cpp-low-level/Dockerfile:25]: I was expecting some CHECKSUM from machine-emulator-tools, so I could check the integrity of the tarball before extracting it.

I've opened an issue for that, and will follow the `curl | tar -f -` path for now.

File: cpp-low-level/Dockerfile [hunk 4]
```diff
@@ -17,13 +17,15 @@ FROM --platform=linux/riscv64 riscv64/ubuntu:22.04
 LABEL io.sunodo.sdk_version=0.1.0
 LABEL io.cartesi.rollups.ram_size=128Mi
 
+ARG MACHINE_EMULATOR_TOOLS_VERSION=0.12.0
 RUN <<EOF
 apt-get update
-apt-get install -y --no-install-recommends busybox-static=1:1.30.1-7ubuntu3
+apt-get install -y --no-install-recommends busybox-static=1:1.30.1-7ubuntu3 curl=7.81.0-1ubuntu1.13 ca-certificates=20230311ubuntu0.22.04.1
```
Comment (tuler) [inline cpp-low-level/Dockerfile:23]: Can you sort it, put ca-certificates before curl

**Review by tuler — COMMENTED** [review]
Looks good, apply fixup

**Comment by tuler** [comment]
LGTM.
`sunodo create` is using the `main` branch, so we must merge this only when `sunodo` version is out.

**Review by tuler — APPROVED** [review]

**Pull request merged**